client/views/userEvent.jade

```diff
@@ -35,7 +35,7 @@ template(name="userEvent")
                 .col-sm-2.space-btm-1.space-top-1
                   if isInRole "admin"
                     .dropdown
-                      button.btn.secondary-btn.btn-block.dropdown-toggle(type="button" data-toggle="dropdown")
+                      button.btn.secondary-btn.btn-block.dropdown-toggle(type="button" data-toggle="dropdown" style="white-space: normal;")
```

Comment:
I would try to avoid the `style=""` attribute and go via CSS with `!important`. Neither is perfect though.